pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/service/admin/UserInfo.java

```diff
@@ -3,26 +3,29 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import lombok.AllArgsConstructor;
 import lombok.Data;
+import org.broadinstitute.dsm.db.dto.user.UserDto;
 
+@AllArgsConstructor
 @Data
 public class UserInfo {
```

Comment:
wondering if this can extend UserDto and add  List<UserRole>

Reply:
The UserDto includes an is_active flag, which is only used internally. IOW, we don't want users setting that flag directly.